### ISSUE TITLE
Duplicated 8080 port in docker-compose file prevents building

### ISSUE THREAD

**Issue by Slater91** [rishikanthc/Scriberr#32]
Hello.
I have just tried deploying Scriberr using Dockge; it failed saying that "Bind for 0.0.0.0:8080 failed: port is already allocated". This seems to stem from the fact that port 8080 is used in both the Scriberr and Pocketbase containers. Removing the port 8080 line from the Scriberr container makes the build succeed. Maybe the default should be changed to e.g. 8081 so there is no conflict?
Thanks a lot for developing Scriberr!

**Comment by rishikanthc** [comment]
Ahh my bad. Forgot to cleanup the unused port. Like you said just need to remove 8080 from the Scriberr docker file. No need to change the default port as 8080 is only for pocketbase. Now that pocketbase is in an external container don't have to worry about it. I'll clean it up.

**Comment by Johnr24** [comment]
your documentation hasnt been cleaned up here, and this issue is still in them

**Comment by rishikanthc** [comment]
Yeah currently in the middle of updating the codebase. This old version has accumulated a lot of issues and I have already moved the app from svelte 4 to svelte 5. So it's a new rewrite and documenting the old one is going to be a hassle now. I'll work on fixing and cleaning up this week and put out a new release that will be a stable release. I apologize for the inconvenience

**Comment by rishikanthc** [comment]
This is no longer an issue as we have moved to a dedicated Postgres instance instead of pocketbase
